html/admin/project/project_assetsBoard.twig

```diff
@@ -49,6 +49,36 @@
         </div>
     </div>
 </div>
+{% if "PROJECTS:PROJECT_ASSETS:EDIT:ASSIGNMENT_STATUS"|instancePermissions %}
+<div class="modal fade" id="scanAssetModal">
+    <div class="modal-dialog modal-lg">
```

Comment:
I'm a bit confused at first glance how to use this modal. 
I've worked it out from reading the code (having to press enter to apply) but a "move" button would be preferred as it is a very clear action and nothing will change until the user has had chance to confirm (UX will be nicer)

If you're using the enter key because that's what the barcode scanner ends with, why not make the textbox below a nice multi-select one from select2 and bulk submit at the end - allows a user to confirm the changes before moving on

Reply:
Have added a button and some explanatory text - reason for not using select2 discussed in a comment below (though it may well be the case that I don't know the right incantations to make it work well!)